src/worker/core/phase/newPhaseBeforeDraft.ts

```diff
@@ -96,6 +96,17 @@ const newPhaseBeforeDraft = async (
 		}
 	}
 
+	if (g.get("challengeThanosMode") && Math.random() < 0.2) {
+		const activePlayers = await idb.cache.players.indexGetAll("playersByTid", [
+			0,
+			Infinity,
+		]);
+		console.log(activePlayers.length);
+		for (let i = 0; i < activePlayers.length / 2; i++) {
+			await player.killOne(conditions);
```

Comment:
Does this result in too many notifications popping up? Would it be better combining them into one?

Reply:
Yeah, you'll get a death notification for each of your players that died but it seems to limit it to only 4 shown at a time, so it doesn't seem worth it to notify for each player.